src/unit-test/RecipesList.test.js

```diff
@@ -1,6 +1,7 @@
 import '../setupTest.js'
 import RecipesList from '../components/RecipesList';
-import { tsExternalModuleReference } from '@babel/types';
+import fetch from '../unit-test/mocks/recipeListFetch';
+global.fetch = fetch;
```

Comment:
same as above, I guess it is needed if it is in all the files...

Reply:
See above